### PR TITLE
UI: HDS adoption update <AlertInline> component to use Hds::Alert

### PR DESCRIPTION
**Pull request by hellobontempo**
Use `Hds::Alert @type="compact"` in the `AlertInline` component so styling is consistent throughout UI.

Screenshot 2023-11-30 at 11 22 50 AM

### CODE REVIEW

File: ui/app/components/auth-saml.hbs [hunk 1]
```diff
@@ -22,8 +22,7 @@
         />
       </div>
       <AlertInline
-        @sizeSmall={{true}}
```
Comment (hellobontempo) [inline ui/app/components/auth-saml.hbs:25]: the class `size-small` no longer exists (functionality have been removed previously?), so remove this arg all together.

File: ui/app/templates/components/secret-create-or-update.hbs [hunk 2]
```diff
@@ -26,12 +26,7 @@
         />
       </p>
       {{#if (get this.validationMessages "path")}}
-        <AlertInline
```
Comment (hellobontempo) [inline ui/app/templates/components/secret-create-or-update.hbs:29]: Screenshot 2023-11-30 at 11 31 12 AM

File: ui/app/templates/components/clients/attribution.hbs [hunk 3]
```diff
@@ -112,17 +112,24 @@
         <Hds::Button @text="Cancel" @color="secondary" {{on "click" F.close}} />
       </Hds::ButtonSet>
       {{#if @upgradeExplanation}}
-        <div class="has-text-grey is-size-8 has-top-padding-m">
-          <AlertInline @type="warning">
-            Your data contains an upgrade.
-            <DocLink
-              @path="/vault/docs/concepts/client-count/faq#q-which-vault-version-reflects-the-most-accurate-client-counts"
+        <Hds::Alert class="has-top-padding-m" @type="compact" @color="warning" as |A|>
```
Comment (hellobontempo) [inline ui/app/templates/components/clients/attribution.hbs:115]: Screenshot 2023-11-30 at 12 59 39 PM

File: ui/app/templates/components/secret-create-or-update.hbs (hunk 2 above)
Comment (Monkeychip) [reply ui/app/templates/components/secret-create-or-update.hbs:29]: Should path be capitalized? Just checked the current look on kv and it's "Path can't be blank."

Comment (Monkeychip) [reply ui/app/templates/components/secret-create-or-update.hbs:29]: I realize this is outside the HDS scope as the message is coming from the model.

Comment (Monkeychip) [reply ui/app/templates/components/secret-create-or-update.hbs:29]: We could use the capitalize template helper.

File: ui/lib/core/addon/components/string-list.hbs [hunk 4]
```diff
@@ -53,15 +53,15 @@
         {{/if}}
       </div>
       {{#if (includes index this.indicesWithComma)}}
-        <Icon class="is-flex-v-centered has-text-highlight" @name="alert-triangle-fill" />
+        <Hds::Alert class="is-flex-v-centered" @type="compact" @color="warning" />
```
Comment (Monkeychip) [inline ui/lib/core/addon/components/string-list.hbs:56]: If we're using compact here, should we use AlertInline instead?

**Review by Monkeychip — APPROVED** [review]
A couple of non-blocking comments. Awesome work on this. It's going to look so good with this now in place!

File: ui/lib/core/addon/components/string-list.hbs (hunk 4 above)
Comment (hellobontempo) [reply ui/lib/core/addon/components/string-list.hbs:56]: This is just so the icon + color to the left of the input matches the alert below, there's no message here.

File: ui/app/templates/components/secret-create-or-update.hbs (hunk 2 above)
Comment (hellobontempo) [reply ui/app/templates/components/secret-create-or-update.hbs:29]: 👍 
Screenshot 2023-11-30 at 4 18 43 PM

File: ui/lib/core/addon/components/string-list.hbs [hunk 5]
```diff
@@ -53,15 +53,11 @@
...
       {{/if}}
     </div>
   {{/each}}
   {{#if this.indicesWithComma}}
```
[the remark this answers is not in the document]
Comment (hellobontempo) [reply ui/lib/core/addon/components/string-list.hbs:60]: updated
Screenshot 2023-11-30 at 4 34 09 PM

File: ui/lib/core/addon/components/string-list.hbs (hunk 4 above)
Comment (hellobontempo) [reply ui/lib/core/addon/components/string-list.hbs:56]: updated screenshot in comment below

**Pull request merged**